packages/server/src/utils/config.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from 'path'
+import * as fs from 'fs'
+
+// should be config via .env or config file
+const logDir = path.join(__dirname, '../../../..', 'logs')
```

Comment:
we can also let user specify their own preferred path to store logs.
For example:
```
const logDir = process.env.LOG_PATH ?? path.join(__dirname, '../../../..', 'logs')
```